### PR TITLE
FindSimilarIncidents - add more options

### PR DESCRIPTION
**Pull request by erezh31**
1. Support OR condition.
2. Add custom query argument.

### CODE REVIEW

File: Scripts/script-FindSimilarIncidents.yml [hunk 1]
```diff
@@ -146,6 +150,12 @@ script: |-
 
       if incident_id:
           query = '(-id:%s) and (%s)' % (incident_id, query)
+
+      if extra_query:
+          query += " and (%s)" % extra_query
+
+      demisto.log("Find similar incidents based on initial query: %s" % query)
```
Comment (meirwah) [inline Scripts/script-FindSimilarIncidents.yml]: you really want to leave this here?

File: Scripts/script-FindSimilarIncidents.yml [hunk 2]
```diff
@@ -368,6 +390,25 @@ args:
...
+  predefined:
+  - AND
+  - OR
+  description: Apply "OR" \ "AND" condition between incident fields. This will apply
+    only for fields with "exact match".
+  defaultValue: AND
+- name: skipMissingValues
+  auto: PREDEFINED
+  predefined:
+  - "yes"
```
Comment (meirwah) [inline Scripts/script-FindSimilarIncidents.yml]: values shouldn't be `AND/OR` ?

Comment (meirwah) [reply Scripts/script-FindSimilarIncidents.yml]: ok my bad

**Comment by meirwah** [comment]
@erezh31 build fails, try to update branch from master

**Comment by erezh31** [comment]
@meirwah all content builds is failing right now, I think some AWS issue

**Comment by meirwah** [comment]
@erezh31 can you update branch and fix conflicts?

**Comment by erezh31** [comment]
@meirwah done, plz review changes

**Pull request merged**